sdk/storage/azure-storage-files-shares/test/share_client_test.cpp

```diff
@@ -189,115 +189,115 @@ namespace Azure { namespace Storage { namespace Test {
-
-    aLease = *m_shareClient->AcquireLease(CreateUniqueLeaseId(), leaseDuration);
-    brokenLease = *m_shareClient->BreakLease();
-    EXPECT_FALSE(brokenLease.ETag.empty());
-    EXPECT_FALSE(brokenLease.LastModified.empty());
-    EXPECT_NE(brokenLease.LeaseTime, 0);
-
-    Files::Shares::BreakShareLeaseOptions options;
-    options.BreakPeriod = 0;
-    m_shareClient->BreakLease(options);
-  }
-
-  TEST_F(FileShareClientTest, SnapshotLease)
-  {
-    std::string leaseId1 = CreateUniqueLeaseId();
-    int32_t leaseDuration = 20;
-    auto snapshotResult = m_shareClient->CreateSnapshot();
-    auto shareSnapshot = m_shareClient->WithSnapshot(snapshotResult->Snapshot);
-    auto aLease = *shareSnapshot.AcquireLease(leaseId1, leaseDuration);
-    EXPECT_FALSE(aLease.ETag.empty());
-    EXPECT_FALSE(aLease.LastModified.empty());
-    EXPECT_EQ(aLease.LeaseId, leaseId1);
-    aLease = *shareSnapshot.AcquireLease(leaseId1, leaseDuration);
-    EXPECT_FALSE(aLease.ETag.empty());
-    EXPECT_FALSE(aLease.LastModified.empty());
-    EXPECT_EQ(aLease.LeaseId, leaseId1);
-
-    auto properties = *shareSnapshot.GetProperties();
-    EXPECT_EQ(properties.LeaseState.GetValue(), Files::Shares::LeaseStateType::Leased);
-    EXPECT_EQ(properties.LeaseStatus.GetValue(), Files::Shares::LeaseStatusType::Locked);
-    EXPECT_EQ(Files::Shares::LeaseDurationType::Fixed, properties.LeaseDuration.GetValue());
-
-    auto rLease = *shareSnapshot.RenewLease(leaseId1);
-    EXPECT_FALSE(rLease.ETag.empty());
-    EXPECT_FALSE(rLease.LastModified.empty());
-    EXPECT_EQ(rLease.LeaseId, leaseId1);
-
-    std::string leaseId2 = CreateUniqueLeaseId();
-    EXPECT_NE(leaseId1, leaseId2);
-    auto cLease = *shareSnapshot.ChangeLease(leaseId1, leaseId2);
-    EXPECT_FALSE(cLease.ETag.empty());
-    EXPECT_FALSE(cLease.LastModified.empty());
-    EXPECT_EQ(cLease.LeaseId, leaseId2);
-
-    auto blobInfo = *shareSnapshot.ReleaseLease(leaseId2);
-    EXPECT_FALSE(blobInfo.ETag.empty());
-    EXPECT_FALSE(blobInfo.LastModified.empty());
-
-    aLease = *shareSnapshot.AcquireLease(CreateUniqueLeaseId(), c_InfiniteLeaseDuration);
-    properties = *shareSnapshot.GetProperties();
-    EXPECT_EQ(Files::Shares::LeaseDurationType::Infinite, properties.LeaseDuration.GetValue());
-    auto brokenLease = *shareSnapshot.BreakLease();
-    EXPECT_FALSE(brokenLease.ETag.empty());
-    EXPECT_FALSE(brokenLease.LastModified.empty());
-    EXPECT_EQ(brokenLease.LeaseTime, 0);
-
-    aLease = *shareSnapshot.AcquireLease(CreateUniqueLeaseId(), leaseDuration);
-    brokenLease = *shareSnapshot.BreakLease();
-    EXPECT_FALSE(brokenLease.ETag.empty());
-    EXPECT_FALSE(brokenLease.LastModified.empty());
-    EXPECT_NE(brokenLease.LeaseTime, 0);
-
-    Files::Shares::BreakShareLeaseOptions options;
-    options.BreakPeriod = 0;
-    shareSnapshot.BreakLease(options);
-
-    EXPECT_THROW(m_shareClient->Delete(), StorageError);
-  }
+  // TEST_F(FileShareClientTest, Lease)
```

Comment:
```suggestion
  TEST_F(FileShareClientTest, DISABLED_Lease)
```

Reply:
This shouldn't be disabled. This should be removed in master with product code due to service not ready.